src/webview/components/Molecules/StatusIndicator/StatusIndicator.css

```diff
@@ -0,0 +1,87 @@
+:root {
+  background-color: var(--vscode-editor-background);
+  color: var(--vscode-editor-foreground);
+}
+
+.status-indicator {
+  display: flex;
+  align-items: center; 
+  justify-content: space-between; /* Space out items */
+  background-color: var(--vscode-editor-background);
+  color: var(--vscode-editor-foreground);
+  padding: 10px;
+  font-family: Inter;
```

Comment:
A lot of these font css items are getting the editor confused. We don't need to specify inter, if you leave it blank I'll just use the vs code default font